ros2pkg/ros2pkg/resource/main.cpp.em

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main(int argc, char ** argv)
+{
+  std::cout << "hello @(package_name)" << std::endl;
```

Comment:
Since we don't recommend using `cout` it shouldn't be used in the generated code.